lib/results.js

```diff
@@ -14,7 +14,7 @@ var nextTick = typeof setImmediate !== 'undefined'
 ;
 
 module.exports = Results;
-inherits(Results, EventEmitter);
+inherits(Results, EventEmitter.EventEmitter || EventEmitter);
```

Comment:
this `||` should happen up at the top, at the declaration site, rather than at the usage site.